openedx/features/course_experience/static/course_experience/js/CourseHome.js

```diff
@@ -4,13 +4,12 @@ export class CourseHome {  // eslint-disable-line import/prefer-default-export
   constructor(options) {
     // Logging for course tool click events
     const $courseToolLink = $(options.courseToolLink);
-    $courseToolLink.on('click', () => {
-      const courseToolName = document.querySelector('.course-tool-link').text.trim().toLowerCase();
+    $courseToolLink.on('click', (event) => {
+      const courseToolName = event.srcElement.text.trim().toLowerCase();
```

Comment:
@HarryRein @robrap I agree that a data attribute seems like the simplest option here.